src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -969,6 +969,11 @@ public FormValidation doCheckName(@QueryParameter final String name, @QueryParam
             if (Boolean.valueOf(isNewCloud) && !CloudNames.isUnique(name)) {
                 return FormValidation.error("Please choose a unique name. Existing clouds: " + Jenkins.get().clouds.stream().map(c -> c.name).collect(Collectors.joining(",")));
             }
+            else if (!Boolean.valueOf(isNewCloud) && !CloudNames.isExistingUnique(name)) {
+                Set<String> uniqueNames = new HashSet<>();
+                Jenkins.get().clouds.forEach(cloud -> {uniqueNames.add(cloud.name);});
+                return FormValidation.error("Existing cloud name is duplicated. Please update to unique names. Existing names: " + uniqueNames);
```

Comment:
How about `This cloud name is not unique. Please choose a unique name and click save. Existing clouds: `